scilpy/segment/voting_scheme.py

```diff
@@ -117,74 +155,57 @@ def _load_bundles_dictionary(self, bundles_filepath):
             bundle = transform_streamlines(streamlines, self.transformation)
             model_bundles_dict[filename] = bundle
 
-            logging.debug('Loaded {0} with {1} streamlines'.format(filename,
-                                                                   len(bundle)))
             if len(bundle) > 5000:
                 logging.warning(
                     '{0} has above 5000 streamlines'.format(filename))
 
         return model_bundles_dict
 
-    def _find_max_in_sparse_matrix(self, bundle_id, min_vote,
-                                   streamlines_wise_vote, bundles_wise_vote):
+    def _find_max_in_sparse_matrix(self, bundle_id, min_vote, bundles_wise_vote):
         """
         Will find the maximum values of a specific row (bundle_id), make
         sure they are the maximum values across bundles (argmax) and above the
         min_vote threshold. Return the indices respecting all three conditions.
-        :param bundle_id, int, indices of the bundles in the dok_matrix
-        :param min_vote, int, minimum value for considering (voting)
-        :param streamlines_wise_vote, dok_matrix, streamlines-wise
-            sparse matrix for voting
-        :param bundles_wise_vote, dok_matrix, bundles-wise
-            sparse matrix for voting
+        :param bundle_id, int, indices of the bundles in the lil_matrix.
```

Comment:
This is more a question on scilpy than a comment on the review, but are we using two different docstring types for public methods/classes and private methods ?

Reply:
This is laziness on my part, but I felt like a full-blown giga docstring would be useless for some functions. So a smaller, simpler version felt appropriate at the time (it was WIP)

I will do it properly in a different PR